lab-kyle/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+let express = require('express')
+let mongoose = require('mongoose')
+let morgan = require('morgan')
+
+let authMiddleware = require('./lib/auth')
+
+const PORT = process.env.PORT || 3000
+const MONGO_URI = process.env.MONGO_URI || 'mongodb://localhost/dev'
+
+mongoose.connect(MONGO_URI)
+mongoose.Promise = Promise
+
+let app = express()
+
+app.use(morgan('dev'))
+
+require('./route/user-routes')(app)
```

Comment:
It's functional with the app passed into the routes module, though more conventional to create a Router object, `let router = express.Router()`, and pass the router in through dep injection.